### PR TITLE
Create test for Radar

### PR DESCRIPTION
**Pull request by A9afon**
Create test for Radar

### CODE REVIEW

File: src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java [hunk 1]
```diff
@@ -122,13 +122,13 @@ void shouldFindByPrimaryAndActiveRadars() {
...
+    Mockito.when(radarRepository.findByPrimaryAndActive(any(boolean.class), any(boolean.class))).thenReturn(radarList);
 
-    radarService.findByPrimaryAndActive(radar.isPrimary(), radar.isActive());
-    // Where is checking?
+    List<RadarDto> radarDtoOptional = radarService.findByPrimaryAndActive(any(boolean.class), any(boolean.class));
+    // Checking for content of primary and active in radarList
+    Assertions.assertFalse(radarDtoOptional.isEmpty());
 
-    Mockito.verify(radarRepository).findByPrimaryAndActive(radar.isPrimary(), radar.isActive());
+    Mockito.verify(radarRepository).findByPrimaryAndActive(any(boolean.class), any(boolean.class));
```
Comment (arufanov) [inline src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java:131]: @A9afon 
* we use here value, not class.

**Review by arufanov — CHANGES_REQUESTED** [review]

File: src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    radar.setRadarType(null);
+    radar.setTitle("Radar title");
+    radar.setDescription("Radar Description");
+    radar.setPrimary(false);
+
+    List<Radar> radarList = List.of(radar);
+    Mockito.when(radarRepository.findByPrimary(any(boolean.class))).thenReturn(radarList);
+
+    Optional<Radar> radarOptional = radarService.findByPrimary(radar.isPrimary());
+    // radarList has list any radar primary=false
```
Comment (arufanov) [inline src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java:111]: @A9afon 
* remove please

File: src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+    radar.setTitle("Radar title");
+    radar.setDescription("Radar Description");
+    radar.setPrimary(true);
+    radar.setActive(true);
+
+    List<Radar> radarList = List.of(radar);
+    Mockito.when(radarRepository.findByPrimaryAndActive(any(boolean.class), any(boolean.class))).thenReturn(radarList);
+
+    List<RadarDto> radarDtoOptional = radarService.findByPrimaryAndActive(radar.isPrimary(), radar.isActive());
+    // Checking for content of primary and active in radarList
```
Comment (arufanov) [inline src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java:131]: @A9afon 
* remove please

**Review by arufanov — CHANGES_REQUESTED** [review]

File: src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+    radar.setRadarType(null);
+    radar.setTitle("Radar title");
+    radar.setDescription("Radar Description");
+    radar.setPrimary(false);
+
+    List<Radar> radarList = List.of(radar);
+    Mockito.when(radarRepository.findByPrimary(any(boolean.class))).thenReturn(radarList);
+
+    Optional<Radar> radarOptional = radarService.findByPrimary(radar.isPrimary());
+    Assertions.assertTrue(radarOptional.isPresent());
```
Comment (arufanov) [inline src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java:110]: @A9afon 
* why do you skip compare with radar?

**Review by arufanov — CHANGES_REQUESTED** [review]

File: src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java (hunk 4 above)
Comment (A9afon) [reply src/test/java/com/a5lab/axion/domain/radar/RadarServiceTests.java:110]: Ok, I'll add.

**Review by arufanov — APPROVED** [review]

**Pull request merged**